### PR TITLE
Cleanup.

### PR DESCRIPTION
**Pull request by nvohra3**
https://github.com/goutamvenkat/ClassMood/issues/61

Tested deletes and gauge resets to make sure they update in DB correctly.

### CODE REVIEW

File: ClassMoodApp/static/tsc/controllers/ClassListController.ts [hunk 1]
```diff
@@ -120,6 +120,31 @@ module ClassMoodApp {
             this.$window.location.href = `/live_lecture/get/${liveLectureId}`;
         }
 
+        public deleteClass(classId: number) {
+            console.log("hi")
```
Comment (krabagobanjo) [inline ClassMoodApp/static/tsc/controllers/ClassListController.ts]: Do we need this? Ditto for various other print statements.

**Review by krabagobanjo — APPROVED** [review]
My only other comment is that Zach pushed out some code fixes that aren't present here, so just take care when merging.

File: ClassMoodApp/static/tsc/controllers/ClassListController.ts (hunk 1 above)
Comment (nvohra3) [inline ClassMoodApp/static/tsc/controllers/ClassListController.ts]: Lol, I'll take those out. My bad.

**Comment by goutamvenkat** [comment]
Approving Pull request

**Review by goutamvenkat — APPROVED** [review]

**Pull request merged**